### PR TITLE
[TR] PIM-4705: Update controller and grid to be able to configure job to launch with quick export

### PR DESCRIPTION
**Pull request by willy-ahva**
| Q                  | A
| ------------------ | ---
| Migration script?  | n
| Behats             | running
| Specs & Static     | ok
| Changelog updated  | n

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Connector/Reader/MassEdit/FilteredProductReader.php [hunk 1]
```diff
@@ -68,12 +68,12 @@ public function read()
         $configuration = $this->getJobConfiguration();
 
         if (null === $configuration) {
-            return null;
+            return;
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Connector/Reader/MassEdit/FilteredProductReader.php]: `null` should be kept here imho

**Comment by phaseinducer** [comment]
little comment otherwise GTM

File: src/Pim/Bundle/EnrichBundle/Connector/Reader/MassEdit/FilteredProductReader.php (hunk 1 above)
Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Connector/Reader/MassEdit/FilteredProductReader.php]: `return;` returns null. So we have an explicit return and we are consistent with the rest of the PIM. That's why I removed it.

File: src/Pim/Bundle/DataGridBundle/Controller/ProductExportController.php [hunk 2]
```diff
@@ -95,9 +97,9 @@ public function indexAction()
     /**
      * Get a user from the Security Context
      *
-     * @return \Symfony\Component\Security\Core\User\UserInterface|null
+     * @return UserInterface|null
      *
-     * @see Symfony\Component\Security\Core\Authentication\Token\TokenInterface::getUser()
+     * @see \Symfony\Component\Security\Core\Authentication\Token\TokenInterface::getUser()
```
Comment (jmleroux) [inline src/Pim/Bundle/DataGridBundle/Controller/ProductExportController.php]: No FQCN

**Comment by jmleroux** [comment]
GTM

File: src/Pim/Bundle/DataGridBundle/Controller/ProductExportController.php (hunk 2 above)
Comment (willy-ahva) [inline src/Pim/Bundle/DataGridBundle/Controller/ProductExportController.php]: it's just for @see not for PHPDoc, don't know what is rule for that.

**Pull request merged**